### PR TITLE
Modified all GUI except for DiseaseAnalyzer to have header images

### PR DESCRIPTION
**Pull request by yifanl27**
Same as title
Added header images, changed text fonts and a bit of formatting so that interfaces are more consistent
Did not modify DiseaseAnalyzer yet since Jenny's most recent pull request hasn't been merged yet, I will be modifying it when it is merged

### CODE REVIEW

**Comment by DavidM0808** [comment]
There are no merge conflicts so I think we can merge it since there are no changes to the main code and it's mostly just formatting.

File: src/main/java/gui/EERPromptGUI.java [hunk 1]
```diff
@@ -203,4 +209,14 @@ private void helperForDisplay(int type){
             }
         }
     }
+
+    private void createUIComponents() {
+        try{
+            headerImg = ImageIO.read(GUIFormatConstants.eerAnalyzerImgFile);
+        }catch (IOException ex){
+            System.out.println("File pathway was not found");
```
Comment (Jennylin331431) [inline src/main/java/gui/EERPromptGUI.java:217]: this is a very small thought, but wondering if "the File pathway was not found" we should put it into the constants class as well? just to make it organized?

File: src/main/java/gui/MealPlanGeneratorGUI.java [hunk 2]
```diff
@@ -183,9 +188,9 @@ public MealPlanGeneratorGUI(String userType) {
             vegetarianCheckBox.setText(MealPlanConstants.VEG);
             numFoodsPrompt.setText(MealPlanConstants.NUM_FOODS);
             invalidInput.setText(Exceptions.INVALID_INPUT);
-            enterButton.setText("Enter");
+            enterButton.setText("ENTER");
             intro.setText(MealPlanConstants.MEALPLAN_INTRO_GUI);
-            returnToMenu.setText("Return to Menu");
+            returnToMenu.setText("RETURN TO MENU");
```
Comment (Jennylin331431) [inline src/main/java/gui/MealPlanGeneratorGUI.java:193]: maybe we should factor out these "enter" and "return to Menu" into the constants as many of these are used throughout?

**Review by Jennylin331431 — COMMENTED** [review]
Mostly looks good to me - added a few comments in terms of organization.

**Pull request merged**